library/src/main/java/com/novoda/downloadmanager/DownloadBatchFactory.java

```diff
@@ -40,7 +40,7 @@ static DownloadBatch newInstance(Batch batch,
             FilePersistence filePersistence = filePersistenceCreator.create();
             DownloadFile downloadFile = new DownloadFile(
                     downloadBatchId,
-                    fileUrl,
```

Comment:
shall we also add a `downloadFileId` here? then we wouldn't need to access to the status for getting it

Reply:
https://github.com/novoda/download-manager/pull/292/commits/4e16f8509a1fed9063ec48db77b76c246e9cf71b